nav2_dwb_controller/dwb_controller/src/dwb_controller.cpp

```diff
@@ -73,6 +75,9 @@ DwbController::on_configure(const rclcpp_lifecycle::State & state)
     node, costmap_ros_->getTfBuffer(), costmap_ros_);
   planner_->on_configure(state);
 
+  get_parameter("controller_frequency", controller_frequency_);
+  RCLCPP_INFO(get_logger(), "Controller frequency set to %.4fHz", controller_frequency_);
```

Comment:
This seems like alot of sig figs for usually an int (20.0000)

Reply:
It was what ROS1 did. I didn't have a good reason to change it. It seemed harmless, and in the rare case someone did use a fractional frequency, it would be reported correctly.